### PR TITLE
Book validation

### PR DESCRIPTION
**Pull request by Karpo99**

### CODE REVIEW

File: src/main/java/store/mybookstore/exception/NoSuchSpecificationProvider.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package store.mybookstore.exception;
+
+public class NoSuchSpecificationProvider extends RuntimeException {
+    public NoSuchSpecificationProvider(String message) {
```
Comment (kozhukhovsky) [inline src/main/java/store/mybookstore/exception/NoSuchSpecificationProvider.java:4]: why empty?

File: src/main/java/store/mybookstore/repository/book/TitleSpecificationProvider.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+import java.util.Arrays;
+import org.springframework.data.jpa.domain.Specification;
+import org.springframework.stereotype.Component;
+import store.mybookstore.model.Book;
+import store.mybookstore.repository.SpecificationProvider;
+
+@Component
+public class TitleSpecificationProvider implements SpecificationProvider<Book> {
+
```
Comment (kozhukhovsky) [inline src/main/java/store/mybookstore/repository/book/TitleSpecificationProvider.java]:
```suggestion

```

File: src/main/resources/db/changelog/db.changelog-master.yaml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+databaseChangeLog:
+  - include:
+      file: db/changelog/changes/01-create-book-table.yaml
```
Comment (kozhukhovsky) [inline src/main/resources/db/changelog/db.changelog-master.yaml]:
```suggestion
      file: db/changelog/changes/01-create-book-table.yaml
      
```

**Review by kozhukhovsky — APPROVED** [review]

File: src/main/java/store/mybookstore/exception/NoSuchSpecificationProvider.java (hunk 1 above)
Comment (Karpo99) [reply src/main/java/store/mybookstore/exception/NoSuchSpecificationProvider.java:4]: What do you mean?

Comment (Karpo99) [reply src/main/java/store/mybookstore/exception/NoSuchSpecificationProvider.java:4]: I see, ok

**Pull request merged**